package-scripts.js

```diff
@@ -266,7 +266,7 @@ module.exports = {
     docs: {
       default: {
         script:
-          'nps docs.prebuild && eleventy && nps docs.postbuild && nps docs.api',
+          'nps docs.prebuild && nps docs.api && eleventy && nps docs.postbuild',
```

Comment:
`nps docs.api` is failed because `_dist` directory doesn't exist.

```
Generating output files...
Finished running in 1.60 seconds.
cp: docs/_dist/api: No such file or directory
The script called "docs.api" which runs "nps docs.preprocess.api && jsdoc -c jsdoc.conf.json && cp LICENSE docs/_dist/api" failed with exit code 1 https://github.com/kentcdodds/nps/blob/v5.9.3/other/ERRORS_AND_WARNINGS.md#failed-with-exit-code
```